packages/text-input/src/TextInput/TextInput.styles.ts

```diff
@@ -387,10 +458,32 @@ export const stateIndicatorStyles: Record<
 
 export const optionalTextBaseStyle = css`
   font-size: 12px;
+  line-height: 12px;
   font-style: italic;
   font-weight: normal;
+  display: flex;
+  align-items: center;
+  > p {
+    margin: 0;
+  }
 `;
 
+// Below numbers are "magic" because they need to account for border width, font line height, etc that would need larger refactoring to make DRY
+export const optionalTextHeight: Record<SizeVariant, string> = {
```

Comment:
Do you need to set the height here? It looks like if i remove the height in the inspector the text is still centered.